applications/tasks/src/kz_notify_resend.erl

```diff
@@ -145,6 +155,9 @@ next() ->
 %% @end
 %%------------------------------------------------------------------------------
 -spec handle_call(any(), kz_term:pid_ref(), state()) -> kz_types:handle_call_ret_state(state()).
+handle_call('trigger_timeout', _From, State) ->
```

Comment:
Should cancel the timer in `State` if it exists

Reply:
oh right, let me do that